x/adminmodule/keeper/proposal_archive_legacy.go

```diff
@@ -0,0 +1,34 @@
+package keeper
+
+import (
+	"github.com/cosmos/admin-module/x/adminmodule/types"
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govv1beta1types "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
+)
+
+func (k Keeper) GetArchivedProposalsLegacy(ctx sdk.Context) []*govv1beta1types.Proposal {
```

Comment:
Not to burden Pasha again - how about create new task to create unit tests for Keeper methods.
It's really easy to make a mistake here with all keys and stuff